### ISSUE TITLE
Info icon content on dashboard is not accessible by keyboard

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#4013]
Info icon is after the 'Search'. When you access the info icon by keyboard, there is no way to get to the link in the content. 

<img width="798" height="108" alt="Image" src="https://github.com/user-attachments/assets/a3c8ce5e-9a2c-48af-9296-f4b7d238af4a" />
